src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -121,10 +121,10 @@ const PastAnalysisTable: React.FC = () => {
 
   const columns = [
     {
-      title: "Study",
+      title: "Status",
       cellFormatters: [expandable]
     },
-    "Study Date", "Patient MRN", "Patient DOB", "Analysis Created"
+    "Study", "Study Date", "Patient MRN", "Patient DOB", "Analysis Created"
```

Comment:
Should the first item be called 'Status'?

Reply:
Wait, scratch that, I see that the 'Status' is the expandable row item above. Please confirm.

Reply:
Yep, needed that for the expandable icon next to 'Status'.